packages/client/src/pages/Game.tsx

```diff
@@ -122,7 +124,7 @@ export const Game = () => {
     };
   }, []);
 
-  const isSpectator = !players.some((player) => player.username === username);
+  const isSpectator = players.length > 0 && !players.some((player) => player.username === username);
```

Comment:
Is this needed because of the initial state?

Reply:
yes, otherwise it won't show Invite player